partials/forms/select.html

```diff
@@ -1,4 +1,4 @@
-<div id="{{id}}-group" class="form-group{{#compound}} form-group-compound{{/compound}}{{#error}} error{{/error}}">
+<div id="{{id}}-group" class="form-group{{#compound}} form-group-compound{{/compound}}{{#error}} error{{/error}} " {{#attributes}} {{attribute}}="{{value}}" {{/attributes}}>
```

Comment:
I think the two spaces before the opening `attributes` tag and the space before the closing `attributes` tags can be removed.